### ISSUE TITLE
Support POST requests

### ISSUE THREAD

**Issue by fakerybakery** [garethbjohnson/gutendex#34]
Hi,
Amazing service! Thank you so much!
Might it be possible to allow POST requests as well?

**Comment by garethbjohnson** [comment]
Hi, I am glad you like it!

Do you mean requests to add new books to the database? If so, then it would be outside of my intended scope for this project, which is to give the Project Gutenberg catalogue, which is already added to the database without POST requests.

**Comment by fakerybakery** [comment]
Hi, sorry, I meant the HTTP POST method to query books.

**Comment by garethbjohnson** [comment]
Hi, OK, why would you list to query with POST? Why not GET?

**Comment by fakerybakery** [comment]
Hi, it's a bit easier to use for search queries

**Comment by garethbjohnson** [comment]
Hi again, I find that POST requests already work. Did you mean that you would like to use the request body to filter results instead of query parameters?

**Comment by fakerybakery** [comment]
Yes, that's what I mean. Not sure if it's too difficult to implement, I could try to make a PR

**Comment by herrAlison** [comment]
I tried everything, but unfortunately without success,

import requests


with requests.session() as session:

 url = 'https://register.fdsinstitut.com/ext/index.php'

 headers = {}
 cookies = {}
 donnees = {}  

 reponse = session.post(url, headers=headers, cookies=cookies, data=donnees)

 print(reponse.text)
